sdk/include/psdk/shobjidl.idl

```diff
@@ -1442,6 +1442,11 @@ cpp_quote("#define CDBOSC_STATECHANGE  0x00000004")
 ]
 interface ICommDlgBrowser2 : ICommDlgBrowser
 {
+cpp_quote("#define CDB2N_CONTEXTMENU_DONE  1")
+cpp_quote("#define CDB2N_CONTEXTMENU_START 2")
+cpp_quote("#define CDB2GVF_SHOWALLFILES  0x00000001")
+cpp_quote("#define CDB2GVF_NOSELECTVERB  0x00000008")
```

Comment:
You don't want to take the opportunity to add the other CDB2GVF flags as well? (even if they aren't yet implemented)

Reply:
comdlg32 does not even implement ICommDlgBrowser2 and all the missing flags are for Vista features we don't have. I could certainly implement CDB2GVF_ISFILESAVE in another PR but not here. CDB2GVF_ALLOWPREVIEWPANE is documented as "not used".

Reply:
OK then let's keep these for now.